templates/ts/api-plugin-from-scratch/README.md

```diff
@@ -0,0 +1,49 @@
+# Overview of API Plugin from New API Template
+
+## Build an API Plugin from a new API with Azure Functions
+
+This app template allows Teams to interact directly with third-party data, apps, and services, enhancing its capabilities and broadening its range of capabilities. It allows Teams to:
+
+- Retrieve real-time information, for example, latest news coverage on a product launch.
+- Retrieve knowledge-based information, for example, my team’s design files in Figma.
+
+## Get started with the template
+
+> **Prerequisites**
+>
+> To run this app template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-cli)
+
```

Comment:
One more thing in Prerequisites is Copilot License:

- [Copilot for Microsoft 365 license](https://learn.microsoft.com/microsoft-365-copilot/extensibility/prerequisites#prerequisites)